x-pack/plugins/ml/public/application/routing/routes/explorer/state_manager.tsx

```diff
@@ -43,38 +47,24 @@ export const ExplorerUrlStateManager: FC<ExplorerUrlStateManagerProps> = ({
   } = useMlKibana();
   const { mlApi } = mlServices;
 
-  const [globalState] = useUrlState('_g');
   const [stoppedPartitions, setStoppedPartitions] = useState<string[] | undefined>();
-  const [invalidTimeRangeError, setInValidTimeRangeError] = useState<boolean>(false);
 
   const timeBuckets = useTimeBuckets(uiSettings);
   const timefilter = useTimefilter({ timeRangeSelector: true, autoRefreshSelector: true });
+  const mlJobService = useMlJobService();
+  const { selectedIds: jobIds, selectedJobs } = useJobSelection(jobsWithTimeRange);
+  const noInfluencersConfigured = getInfluencers(mlJobService, selectedJobs).length === 0;
 
-  const { jobIds } = useJobSelection(jobsWithTimeRange);
   const selectedJobsRunning = jobsWithTimeRange.some(
-    (job) => jobIds.includes(job.id) && job.isRunning === true
+    (job) => jobIds?.includes(job.id) && job.isRunning === true
   );
 
   const anomalyExplorerContext = useAnomalyExplorerContext();
-  const { explorerService } = anomalyExplorerContext;
-  const explorerState = useObservable(anomalyExplorerContext.explorerService.state$);
+  const [explorerState, setExplorerState] = useState<ExplorerState>(getExplorerDefaultState());
 
   const refresh = useRefresh();
   const lastRefresh = refresh?.lastRefresh ?? 0;
 
-  // We cannot simply infer bounds from the globalState's `time` attribute
```

Comment:
I think it’s no longer used, as I couldn’t set `globalState.time.mode` to an `invalid` state.